.github/workflows/build-and-deploy-notifications.yml

```diff
@@ -67,7 +67,7 @@ jobs:
   deploy:
     name: Deploy to ${{ matrix.environment }} environment
     runs-on: ubuntu-latest
-    if: github.ref == 'refs/heads/main' && github.event_name == 'push'
+    #if: github.ref == 'refs/heads/main' && github.event_name == 'push'
```

Comment:
Commented out lines should either be removed or uncommented before PRs are raised

Reply:
removed